helmfile.d/values/falco/falco-common.yaml.gotmpl

```
@@ -164,9 +164,18 @@ customRules:
           (container.image.repository = quay.io/calico/node and proc.name = calico-node) or
           (container.image.repository = registry.k8s.io/dns/k8s-dns-node-cache and proc.name = node-cache)
         )
+
+    {{- if and (eq .Values.calicoAccountant.enabled true) (eq .Values.calicoAccountant.backend "nftables") }}
```

Should be the same, right :sweat_smile: ?
```suggestion
    {{- if and .Values.calicoAccountant.enabled (eq .Values.calicoAccountant.backend "nftables") }}
```